frontend/apps/filemanager/filemanager.lua

```diff
@@ -533,6 +543,30 @@ function FileManager:setHome(path)
     return true
 end
 
+function FileManager:openRandomFile(dir)
+    local random_file = DocumentRegistry:getRandomFile(dir, false)
+    if random_file then
+        UIManager:show(MultiConfirmBox:new {
+            text = T(_("Do you want to open %1?"), require("ffi/util").basename(random_file)),
```

Comment:
Perhaps the name is a slight vestige, but notice that around line 30… `local util = require("ffi/util")` ;-)